core/server/web/api/v2/admin/middleware.js

```diff
@@ -5,8 +5,9 @@ const shared = require('../../../shared');
  * Authentication for private endpoints
  */
 module.exports.authAdminAPI = [
+    auth.authenticate.authenticateAdminApiKey,
     auth.authenticate.authenticateAdminAPI,
-    auth.authorize.authorizeAdminAPI,
+    auth.authorize.requiresAuthorizedUserOrApiKey,
```

Comment:
@gargol I would for the beginning disconnect these middlewares and re-enable them later again.

Reply:
yes, definitely  both `auth.authenticate.authenticateAdminApiKey` and `auth.authorize.requiresAuthorizedUserOrApiKey` should not land in master for now

Reply:
👍 let's push a quick commit if we have merge the permission PRs and rebased against master

Reply:
👍